rskj-core/src/test/java/co/rsk/peg/RegisterBtcTransactionIT.java

```diff
@@ -32,58 +34,108 @@
 
 public class RegisterBtcTransactionIT {
 
+    public static final Coin BTC_TRANSFERRED = Coin.COIN;
```

Comment:
nit: add one space bellow to separate sections (i.e. constants, instance variables, etc)